packages/diagrams/lib/src/cupertino_colors.dart

```diff
@@ -0,0 +1,485 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/cupertino.dart';
+
+import 'diagram_step.dart';
+
+const String _basic_colors = 'cupertino_basic_colors';
+const String _active_colors = 'cupertino_active_colors';
+const String _system_colors_1 = 'cupertino_system_colors_1';
+const String _system_colors_2 = 'cupertino_system_colors_2';
+const String _system_colors_3 = 'cupertino_system_colors_3';
+const String _label_colors = 'cupertino_label_colors';
+const String _background_colors = 'cupertino_background_colors';
+
+List<Map<String, Color>> basicColors = <Map<String, Color>>[
+  <String, Color>{'white': CupertinoColors.white},
+  <String, Color>{'black': CupertinoColors.black},
+  <String, Color>{'lightBackgroundGray': CupertinoColors.lightBackgroundGray},
+  <String, Color>{
+    'extraLightBackgroundGray': CupertinoColors.extraLightBackgroundGray
+  },
+  <String, Color>{'darkBackgroundGray': CupertinoColors.darkBackgroundGray},
+  <String, Color>{'inactiveGray': CupertinoColors.inactiveGray},
+  <String, Color>{'destructiveRed': CupertinoColors.destructiveRed},
```

Comment:
while these aren't "active", they feel like they belong with the three active colors, and were also introduced in iOS 13. Consider moving there there and renaming "Active colors"